Profile first screen

Maximus135 opened the pull request:
Added fixes for comments and checked how this component work.

Maximus135 committed:
profile first screen

gohabereg commented on src/screens/UnauthorizedProfile.tsx:
```diff
@@ -0,0 +1,73 @@
+import styled from 'styled-components/native'
+import PrimaryButton from '../atoms/PrimaryButton'
+
+const StyledView = styled.View`
```
Все styled переменные лучше перенести под код компонента, так удобнее

gohabereg replied:
Имеет смысл называть styled компоненты более осмысленно, типа `Container`, `ButtonLabel` и тд

Maximus135 replied:
поправил

gohabereg commented on src/screens/UnauthorizedProfile.tsx:
```diff
@@ -0,0 +1,73 @@
+  )
+}
+
+const styles = StyleSheet.create({
```
Не очень понял зачем мешать styled-components и StyleSheet, по идее все эти правила можно в styled-component перенести просто

Maximus135 replied:
Не нашел другой возможности накинуть box-shadow так-как react native не умеет так) Возможно, есть варианты как исправить

gohabereg commented on src/screens/UnauthorizedProfile.tsx:
```diff
@@ -0,0 +1,73 @@
+  return (
+    <StyledView>
+      <Modal animationType='slide' transparent visible={modalVisible}>
+        <View>
```
Это кажется лишний контейнер

Maximus135 replied:
убрал

Maximus135 committed:
fix CRLF to LF

gohabereg commented on .eslintrc.json:
```diff
@@ -31,7 +31,7 @@
   "rules": {
     "linebreak-style": ["error", "unix"],
-    //    "linebreak-style": ["error", "windows"],
+    // "linebreak-style": ["error", "windows"],
```
Можно выпилить вообще эту строчку

Maximus135 replied:
выпилил

Maximus135 commented:
<Modal блокирует все элементы и нет возможность переключатся между категориями. Пока не нашел решение данной проблемы.

Maximus135 committed:
first screen

add fixes for modal window, remove <Madal> element because it will not with our navbar menu.

Maximus135 commented:
> <Modal блокирует все элементы и нет возможность переключатся между категориями. Пока не нашел решение данной проблемы.

Исправлено
